tools/missing-test-detector/changed_test.go

```diff
@@ -0,0 +1,204 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func TestResourceMapChanges(t *testing.T) {
```

Comment:
Done.
For the purposes of missing test detection, deleted fields and resources are not considered changes because they do not require tests covering them.